domino/routes.py

```diff
@@ -105,3 +105,17 @@ def deployment_version(self):
     # App URLs
     def app_publish(self):
         return self._build_project_url_private_api() + '/nb/startSession'
+
+    # Datasets URLs
+    def datasets_list(self, project_id):
+        if project_id is None:
+            return self.host + '/dataset'
+        else: 
+            return self.host + '/dataset?projectId=' + str(project_id)
+
+    def datasets_create(self): 
+        return self.host + '/dataset'
+
+    def datasets_details(self, dataset_id, dataset_name=None, dataset_description=None): 
```

Comment:
parameter `dataset_name` and `dataset_description` are passed but not used, please remove them